src/Resources/config/serializer/Enqueue.MessengerAdapter.EnvelopeItem.InteropMessageStamp.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="Enqueue\MessengerAdapter\EnvelopeItem\InteropMessageStamp" exclusion-policy="ALL">
+        <property name="message" type="mixed" expose="true" />
```

Comment:
I don't think JMS supports this type

Reply:
You mean type mixed? I tested it in my own installation, and it works.

Reply:
just talked to @goetas on Symfony Slack, jms doesn't support `type="mixed"`

Reply:
Alright. The problem is that the implementation of message inside the InteropMessageStamp is against an interface. So the actual type depends on the implementation. Therefore we have to remove that stamp configuration as JMS does also not support serialization of interfaces and each project therefore has to implement its own serializer configuration. Do you agree?